boards/xtensa/esp32s3/esp32s3-devkit/configs/fastboot/defconfig

```diff
@@ -0,0 +1,11 @@
+#include "../usb_device/defconfig"
```

Comment:
Yes, but I couldn't find any other usage of such syntax on NuttX (and I don't think this kind of syntax is generated by `make savedefconfig`, so I prefer not to mix syntaxes.

Any ideas about it @cederom , @acassis and @jerpelea ?